decidim-proposals/spec/system/admin/highlighted_proposal_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Highlighted proposal", type: :system do
+  let(:admin) { create :user, :admin, :confirmed }
+  let(:organization) { admin.organization }
+  let!(:participatory_process) { create(:participatory_process, organization: organization) }
+  let!(:proposal_component) { create(:proposal_component, participatory_space: participatory_process) }
+  let!(:proposals) do
+    [
+      create(:proposal, :not_answered, id: 1, created_at: 9.days.ago, published_at: 9.days.ago, component:
+        proposal_component),
+      create(:proposal, :not_answered, id: 2, created_at: 5.days.ago, published_at: 5.days.ago, component:
+        proposal_component),
+      create(:proposal, :with_answer, id: 3, created_at: 9.days.ago, published_at: 9.days.ago, component:
+        proposal_component)
+    ]
+  end
+
+  before do
+    switch_to_host(organization.host)
```

Comment:
```suggestion
    switch_to_host(organization.host)
    allow(Decidim::Proposals::Proposal).to receive(:unanswered_proposals_overdue).and_return(days_overdue)
```
This is a trick we use quite often, allows Rspec to substitue one method that you want to change to test different situations.